librz/core/canalysis.c

```diff
@@ -6914,7 +6914,18 @@ RZ_IPI bool rz_core_analysis_everything(RzCore *core, bool experimental, char *d
 		rz_print_rowlog_done(core->print, oldstr);
 		rz_core_analysis_autoname_all_golang_fcns(core);
 		oldstr = rz_print_rowlog(core->print, "Analyze all flags starting with sym.go. (aF @@f:sym.go.*)");
-		rz_core_cmd0(core, "aF @@f:sym.go.*");
+		RzAnalysisFunction *fcn;
+		RzListIter *iter;
+		if (core->analysis) {
```

Comment:
`@@f` is iteration over flags, not functions.

Reply:
I checked the actual code of `@@f` foreach in `librz/core/cmd.c` and copied the logic.

Reply:
That is the oldshell one. We merged most of `@@` and `@@@` and now there is IMO less confusion. Just do `@@?` to see what we support. In particular, you can find the `@@f:` code in `iter_flags_command`.

Reply:
Ok, removed that change from the PR, will deal with it later.